dpnp/backend/mkl_wrap_rng.cpp

```diff
@@ -23,39 +23,29 @@
 // THE POSSIBILITY OF SUCH DAMAGE.
 //*****************************************************************************
 
-#include <ctime>
-#include <iostream>
-#include <vector>
-
 #include <backend_iface.hpp>
 #include "backend_utils.hpp"
 #include "queue_sycl.hpp"
 
 namespace mkl_rng = oneapi::mkl::rng;
 
+// TODO:
+// add mean and std params ?
 template <typename _DataType>
 void mkl_rng_gaussian(void* result, size_t size)
 {
     if (!size)
     {
         return;
     }
-
     _DataType* result1 = reinterpret_cast<_DataType*>(result);
 
-    // TODO:
-    // choose engine as is in numpy
-    // seed number
-    size_t seed = std::time(nullptr);
-    mkl_rng::philox4x32x10 engine(DPNP_QUEUE, seed);
-
     const _DataType mean = _DataType(0.0);
     const _DataType stddev = _DataType(1.0);
 
     mkl_rng::gaussian<_DataType> distribution(mean, stddev);
     // perform generation
-    mkl_rng::generate(distribution, engine, size, result1);
-
+    auto event_out = mkl_rng::generate(distribution, DPNP_RNG_ENGINE, size, result1);
```

Comment:
What is the type of `event_out`?
avoid using `auto` as much as possible.
this variable is not used - why we need it?
if it is `sycl::event` - use it to `.wait()` instead `queue.wait()`

Reply:
for auto var type is deduced from the initializer. most of mkl sycl funcs for usm mem return sycl::event. But thank you for the comment I can redesign it.